### PR TITLE
fix(new-releases): use `getContents` for `getArtistsList`

### PR DESCRIPTION
**Pull request by machinemessiah**
Restore `getArtistList` functionality by using `LibraryAPI.getContents` to perform the same operation that was previously handled by `LibraryAPI.getArtists` which no longer exists.

### CODE REVIEW

**Review by machinemessiah — COMMENTED** [review]
Could be cleaner, but it works.

File: CustomApps/new-releases/index.js [hunk 1]
```diff
@@ -269,8 +269,15 @@ class Grid extends react.Component {
...
-	const base = await Spicetify.Platform.LibraryAPI.getArtists();
-	const artists = await Spicetify.Platform.LibraryAPI.getArtists({ limit: base.totalLength });
+	const config = {
+		filters: ["1"],
+		sortOrder: ["0"],
+		textFilter: "",
+		offset: 0,
+		limit: 0,
+	};
+	config.limit = (await Spicetify.Platform.LibraryAPI.getContents(config)).totalLength;
```
Comment (rxri) [inline CustomApps/new-releases/index.js:279]:
```suggestion
		limit: (await Spicetify.Platform.LibraryAPI.getContents(config))?.totalLength ?? 0,
	};
```

**Review by rxri — CHANGES_REQUESTED** [review]

File: CustomApps/new-releases/index.js (hunk 1 above)
Comment (machinemessiah) [reply CustomApps/new-releases/index.js:279]: Cleaner for sure!
Not sure why I didn't do that to begin with...

**Comment by Delusoire** [comment]
and where are the comments mate?

**Comment by rxri** [comment]
Maybe post your review first then ask where are the comments. I already told you so in discord dms.

**Pull request merged**